code/game/objects/effects/spawners/random/random.dm

```diff
@@ -102,10 +102,20 @@
 		for(var/turf/turf_in_view in view(radius, get_turf(src)))
```

Comment:
The comment for this proc is very unhelpful please update it to describe what it says, returns, and what its arg does.
Also preferably make radius an early return